### PR TITLE
Add README for adding selenium tests #33

### PR DESCRIPTION
**Pull request by ChinmayDharwad24**
# Description
Summarize the change.

Fixes #<issue-number> (link all the GitHub issues this addresses)

Add README for adding selenium tests #33

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: Selenium_Test_README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Addding Selenium Tests
+
+## Overview
+This repository contains a suite of Selenium tests designed to validate the functionality of a web application. Selenium is a powerful tool for automating browser interactions, and these tests are intended to ensure the robustness and correctness of the application under different scenarios.
+
+## Prerequisites
+Before running the Selenium tests, make sure you have the following prerequisites installed on your machine:
+
+* **Node.js**: Ensure you have Node.js installed, as it is required for running JavaScript-based Selenium tests.
```
Comment (nbhavana) [inline Selenium_Test_README.md:9]: Can you add command to install node js via terminal?

**Comment by nbhavana** [comment]
@ChinmayDharwad24 is this PR ready to review?

File: Selenium_Test_README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+```sh
+npm install --save selenium-webdriver chromedriver
+```
+
+For Firefox:
+```sh
+npm install --save selenium-webdriver geckodriver
+```
+
+* Running the Tests
```
Comment (AndrewHess) [inline Selenium_Test_README.md:29]: nit: Make "Running the Tests" a section instead of a bullet point

File: Selenium_Test_README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+1. Clone the Repository
+Clone this repository to your local machine:
```
Comment (AndrewHess) [inline Selenium_Test_README.md:31]: nit: Github isn't rendering this nicely (it's concatenating the lines), so you should change it so it looks better.

Same comment for lines 38-39 and 46-47

**Review by AndrewHess — COMMENTED** [review]
Why does this PR update Go code? If those changes aren't needed for this PR, you should `git revert` them

**Comment by nbhavana** [comment]
@ChinmayDharwad24 any update on this PR?

**Comment by AndrewHess** [comment]
Closing since there's been no recent updates

**Pull request closed without merging**